Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -7,42 +7,32 @@
 use Doctrine\Bundle\MongoDBBundle\Loader\SymfonyFixturesLoaderInterface;
 use Doctrine\Bundle\MongoDBBundle\ManagerRegistry;
 use Doctrine\Common\DataFixtures\Executor\MongoDBExecutor;
-use Doctrine\Common\DataFixtures\Loader;
 use Doctrine\Common\DataFixtures\Purger\MongoDBPurger;
 use RuntimeException;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ConfirmationQuestion;
 use Symfony\Component\Console\Style\SymfonyStyle;
-use Symfony\Component\HttpKernel\KernelInterface;
 
-use function class_exists;
 use function implode;
 use function sprintf;
-use function trigger_deprecation;
 
 /**
  * Load data fixtures from bundles.
  */
 class LoadDataFixturesDoctrineODMCommand extends DoctrineODMCommand
 {
-    public function __construct(?ManagerRegistry $registry = null, ?KernelInterface $kernel = null, private ?SymfonyFixturesLoaderInterface $fixturesLoader = null)
+    public function __construct(ManagerRegistry $registry, private SymfonyFixturesLoaderInterface $fixturesLoader)
     {
         parent::__construct($registry);
     }
 
```

Comment:
The command is removed in the DI extension instead of being disabled, so the code is more centralized and the dependency to the loader can be required in the constructor.